acra/configuring-maintaining/general-configuration/acra-server.md

```diff
@@ -17,21 +17,19 @@ weight: 3
   Enable TLS to encrypt transport between AcraConnector and AcraServer.
   Default is `false` which means "use SecureSession instead".
 
-* `--acraconnector_transport_encryption_disable=`
+* `--acraconnector_transport_encryption_disable={true|false}`
 
   Use raw transport (tcp/unix socket) between AcraTranslator and client app.
   It turns off reading trace from client app's side which usually sent by AcraConnector.
   Default is `false`.
 
 * `--acrastruct_injectedcell_enable={true|false}`
 
-  Acrastruct may be injected into any place of data cell.
-  Default is `false`.
+  AcraStruct may be injected into any place of data cell (**deprecated**, ignored, AcraServer always works in Injected Cell mode).
```

Comment:
```suggestion
  AcraStruct may be injected into any place of data cell (**deprecated since 0.90.0**, ignored, AcraServer always works in Injected Cell mode).
```